cmd/cp-url-syntax.go

```diff
@@ -30,9 +31,6 @@ import (
 
 func checkCopySyntax(ctx context.Context, cliCtx *cli.Context, encKeyDB map[string][]prefixSSEPair, isMvCmd bool) {
 	if len(cliCtx.Args()) < 2 {
-		if isMvCmd {
-			showCommandHelpAndExit(cliCtx, 1) // last argument is exit code.
-		}
```

Comment:
if this is removed remove `isMvCmd bool` argument as well.

Reply:
isMvCmd is used for some error messages, like "Could not guess type of "+operation+" operation", should these be changed to "Could not guess type of move/copy operation" or something else?